pkg/controller/endpointslicemirroring/endpointslicemirroring_controller.go

```diff
@@ -38,10 +38,10 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
+	endpointsliceutil "k8s.io/endpointslice/util/endpointslice"
```

Comment:
It's unfortunate that the EndpointSlice reconciler library is not sufficiently generic to be used by the EndpointSlice Mirroring controller. Although it's out of scope for this PR, it would probably be good to document somewhere, either in the EndpointSlice mirroring controller or in the new EndpointSlice staging lib why that is. I believe it comes down to the fact that the reconciler in the EndpointSlice mirroring controller has a 1:1 mapping between Service/Endpoints and EndpointSlice, which results in a simpler implementation that the EndpointSlice staging lib. 

Now that the code is in a place that it _could_ be shared, we probably want to at least describe why it's not currently shared, and state that contributions to move towards the shared code being used by the mirroring controller would be welcome.

Reply:
not sure it is the right place, but added above to the README.